src/main/ts/Part2Ex4FP.ts

```diff
@@ -120,12 +125,17 @@ const getOrElse1 = <A> (oa: Optional<A>, other: A): A =>
 // Hang on - that looks familiar. The function we pass as the "some" case is the identity function.
 
 // TODO: write a version of getOrElse1 using Fun.identity.
+export const getOrElse2 = <A> (oa: Optional<A>, other: A): A =>
+  oa.fold(
+    () => other,
+    (a) => Fun.identity(a)
```

Comment:
The `Fun.identity` function can be ETA reduced. eg:
```suggestion
    Fun.identity
```

Reply:
Yes, why I didn't do that. That's the whole point of `identity` hehe.